### PR TITLE
Fix a typos, punctuation, and grammatical errors in documentation.

### PR DESCRIPTION
**Pull request by keep94**

### CODE REVIEW

**Comment by keep94** [comment]
I have questions about the following items:

    - In cloud_integration_app_dynamics.md, should we use the word ingestion? or injection? in the argument reference
    - In cloud_integration_ec2.md should we use EC2 or Metrics+
    - In cloud_integration_ec2.md the hostname_tags description missing
    - In dashboard.md, in the argument reference, they refer to "pills" is this correct?

File: docs/resources/alert.md [hunk 1]
```diff
@@ -34,28 +34,28 @@ The following arguments are supported:
...
 * `threshold_targets` - (Optional, `THRESHOLD` alerts only) Targets for severity
 * `additional_information` - (Optional) User-supplied additional explanatory information for this alert.
-Useful for linking runbooks, migrations...etc
+Useful for linking runbooks, migrations, etc.
 * `display_expression` - (Optional) A second query whose results are displayed in the alert user
 interface instead of the condition query.  This field is often used to display a version
 of the condition query with Boolean operators removed so that numerical values are plotted.
 * `resolve_after_minutes` - (Optional) The number of consecutive minutes that a firing series matching the condition
-query must evaluate to "false" (zero value) before the alert resolves.  When unset, this default sto
+query must evaluate to "false" (zero value) before the alert resolves.  When unset, this default to
```
Comment (Margarita-Staneva) [inline docs/resources/alert.md:55]: When unset, this defaults to

File: docs/resources/alert_target.md [hunk 2]
```diff
@@ -33,19 +33,19 @@ resource "wavefront_alert_target" "test_target" {
 
 The following arguments are supported:
 
-* `name` - (Required) The name of the alert target as it is displayed in wavefront
+* `name` - (Required) The name of the alert target as it is displayed in Wavefront
```
Comment (Margarita-Staneva) [inline docs/resources/alert_target.md:36]: Please add a full stop at the end of the sentence.

File: docs/resources/dashboard.md [hunk 3]
```diff
@@ -73,161 +73,161 @@ resource "wavefront_dashboard" "test_dashboard" {
...
-* `fixed_legend_enabled` - (Optional) Whether to enable a fixed tabular legend adjacent to the chart 
-* `fixed_legend_use_raw_stats` - (Optional) If `true`, the legend uses non-summarized stats instead of summarized
-* `fixed_legend_position` - (Optional)  Where the fixed legend should be displayed with respect ot the chart.
-Valid options are `RIGHt`, `TOP`, `LEFT`, `BOTTOM`
-* `fixed_legend_display_stats` - (Optional) For a chart with a fixed legend, a list of statistics to display in the legend
+reenable it when the ctrl-key is pressed).
+* `fixed_legend_enabled` - (Optional) Whether to enable a fixed tabular legend adjacent to the chart.
+* `fixed_legend_use_raw_stats` - (Optional) If `true`, the legend uses non-summarized stats instead of summarized.
+* `fixed_legend_position` - (Optional)  Where the fixed legend should be displayed with respect to the chart.
+Valid options are `RIGHt`, `TOP`, `LEFT`, `BOTTOM`.
```
Comment (Margarita-Staneva) [inline docs/resources/dashboard.md:171]: `RIGHt` should be `RIGHT` I think

**Review by Margarita-Staneva — COMMENTED** [review]
I have some minor comments. Rest looks good!

File: docs/resources/alert.md (hunk 1 above)
Comment (keep94) [reply docs/resources/alert.md:55]: done.

File: docs/resources/dashboard.md (hunk 3 above)
Comment (keep94) [reply docs/resources/dashboard.md:171]: Done.

**Comment by cielowu** [comment]
> I have questions about the following items:
> 
> ```
> - In cloud_integration_app_dynamics.md, should we use the word ingestion? or injection? in the argument reference
> - In cloud_integration_ec2.md should we use EC2 or Metrics+
> - In cloud_integration_ec2.md the hostname_tags description missing
> - In dashboard.md, in the argument reference, they refer to "pills" is this correct?
> ```

I'm not familiar with these context, I guess @Margarita-Staneva or Renate is the best person for these questions

**Comment by Margarita-Staneva** [comment]
> > I have questions about the following items:
> > ```
> > - In cloud_integration_app_dynamics.md, should we use the word ingestion? or injection? in the argument reference
> > - In cloud_integration_ec2.md should we use EC2 or Metrics+
> > - In cloud_integration_ec2.md the hostname_tags description missing
> > - In dashboard.md, in the argument reference, they refer to "pills" is this correct?
> > ```
> 
> I'm not familiar with these context, I guess @Margarita-Staneva or Renate is the best person for these questions

- I think the correct term is ingestion.
- About EC2: We have three AWS integrations: CloudWatch, CloudTrail, and Metric+. Metrics+ retrieves additional metrics using AWS APIs other than CloudWatch. Data include EBS volume data and EC2 instance metadata like tags. 

Not sure who can answer whether EC2 should be updated to Metrics+. Maybe ask someone from our Integrations team (for example Harish)?
- About the description of `hostname_tags`, I'd change the first sentence to: (Optional) A list of AWS instance tags to use as the source name in a series. Rest looks good.
- I don't understand what is meant by "pills" in display_section_table_of_contents. I find the whole description confusing "Whether the "pills" quick-linked the sections of the dashboard are displayed by default when the dashboard is shown." 

What happens when this argument is set to true?

**Comment by keep94** [comment]
Hi Margarita.  Thanks for looking at my PR again.

1. I am using ingestion per your suggestion
2. Regarding the EC2 vs. Metrics+ question, would you happen to know Harish's last name?
3. I changed the description of hostname_tags per your suggestion.
4. I don't understand "pills" either in the display_section_table_of_contents.  The term "pills" appears in the description of "display_section_table_of_contents" in the swagger.json file too, and swagger.json is the ground truth for the wavefront APIs.  I plan to ask Harish about it tomorrow. If he doesn't have an answer, I think we should just leave it the way it is.

**Pull request merged**